eras/conway/test-suite/cardano-ledger-conway-test.cabal

```
@@ -30,19 +30,19 @@ library
 
     build-depends:
         base >=4.14 && <4.19,
-        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
+        cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} ^>=1.6,
         cardano-ledger-alonzo-test,
-        cardano-ledger-babbage >=1.3 && <1.7,
-        cardano-ledger-babbage-test,
-        cardano-ledger-binary >=1.0,
+        cardano-ledger-babbage ^>=1.6,
+        cardano-ledger-babbage-test ^>=1.1.1.9,
+        cardano-ledger-binary ^>=1.3,
```

Bot necessary. There is nothing in `cardano-ledger-conway-test` that depends on the changes in this PR, except for version changes, which only require relaxing of some upper bounds
```suggestion
        cardano-ledger-babbage >=1.3 && <1.7,
        cardano-ledger-babbage-test >=1.1.1,
        cardano-ledger-binary >=1.0,
```

`cardano-ledger-babbage-test >=1.1.1` was moved from `cardano-ledger-conway-test` section